### PR TITLE
Add Sentry

### PR DESCRIPTION
**Pull request by ehanson8**
#### What does this PR do?
* Install sentry-sdk dependency
* Add SENTRY_DSN parameter to config.py
* Add function to config.py for verification of integration

#### How can a reviewer manually see the effects of these changes?
Verify that the project has been created for our org on https://sentry.io/ and that the integration works by viewing the previously triggered exception

#### What are the relevant tickets?
* https://mitlibraries.atlassian.net/browse/DLSPP-152

#### Requires Database Migrations?
NO

#### Includes new or updated dependencies?
YES

### CODE REVIEW

File: awd/config.py [hunk 1]
```diff
@@ -58,3 +61,12 @@
     LOG_SOURCE_EMAIL = os.getenv("LOG_SOURCE_EMAIL")
     LOG_RECIPIENT_EMAIL = os.getenv("LOG_RECIPIENT_EMAIL")
     RETRY_THRESHOLD = os.getenv("RETRY_THRESHOLD")
+    SENTRY_DSN = os.getenv("SENTRY_DSN")
+
+
+def check_sentry():
+    if SENTRY_DSN:
```
Comment (osmandin) [inline awd/config.py:68]: What is the intent behind this? If SENTRY_DSN is absent, line 71 duly reports that.

**Review by osmandin — CHANGES_REQUESTED** [review]
Please refer to the comment. Thanks.

File: awd/config.py (hunk 1 above)
Comment (ehanson8) [reply awd/config.py:68]: This is called when you need to check that the actual connection to Sentry is receiving exception. It is just being stored for those situations, this is happening in the `alma-scripts` and `dspace-submission-service`repos as well

Comment (ehanson8) [reply awd/config.py:68]: It is not regularly called, it is there in for troubleshooting purposes, see the last comment here https://github.com/MITLibraries/alma-scripts/pull/33

Comment (ehanson8) [reply awd/config.py]: Removed

**Review by osmandin — APPROVED** [review]

**Pull request merged**